web-console/src/views/query-view/run-button/run-button.tsx

```diff
@@ -133,6 +140,11 @@ export class RunButton extends React.PureComponent<RunButtonProps, RunButtonStat
             onQueryContextChange(setUseCache(queryContext, !useCache));
           }}
         />
+        <MenuItem
+          icon={IconNames.PROPERTIES}
```

Comment:
please let me know if there is a cooler icon

Reply:
Good choice!